app/routes.py

```diff
@@ -1,2 +1,64 @@
-from flask import Blueprint
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, abort, make_response, request
 
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+
+def validate_model(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        abort(make_response(jsonify({"message":f"{cls.__name__} {model_id} invalid"}), 400))
+    
+    model = cls.query.get(model_id)
+    if not model:
+        abort(make_response(jsonify({"message":f"{cls.__name__} {model_id} not found"}), 404))
+
+    return model
+
+
+
+@planets_bp.route("", methods=["POST"])
+def create_planet():
+    request_body = request.get_json()
+    new_planet = Planet.from_dict(request_body)
+    db.session.add(new_planet)
+    db.session.commit()
+
+    return make_response(jsonify(f"Planet {new_planet.name} created successfully."), 201)
```

Comment:
When you created your Planet class, you made each of the attributes non-nullable, which means that each one needs to be present when an instance of that class is created! With that in mind, it might be a good idea to include some level of error handling when your planet is created in case any of the attributes aren't present!